src/utils/backend/news/form.ts

```diff
@@ -46,9 +52,108 @@ export async function getForm(
 }
 
 export async function sendForm(
-  form: { id: number; value: string | number | string[] | File | null }[]
-): Promise<BackendReturn<FormQuestionsTable[]>> {
-  // TODO: Send form data to Supabase
+  formID: number,
+  formAnswer: { id: number; value: string | number | string[] | File | null }[],
+  sendAs?: number
+): Promise<
+  BackendReturn<
+    BackendReturn<FormFieldValueTable[] | FormFieldValueTable | null>[]
+  >
+> {
```

Comment:
I’m not sure what this means…?

Reply:
Which part? as in what is this for or literally what is this code doing

Reply:
The return type. This part:
```ts
<
  BackendReturn<
    BackendReturn<FormFieldValueTable[] | FormFieldValueTable | null>[]
  >
>
```
Why are there 2 `BackendReturn`s?

Reply:
Good Question

Reply:
Fuck